azure-monitor-opentelemetry/azure/monitor/opentelemetry/_vendor/v0_40b0/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -242,18 +415,23 @@ def set_status_code(span, status_code):
 
 
 def get_default_span_details(scope: dict) -> Tuple[str, dict]:
-    """Default implementation for get_default_span_details
+    """
+    Default span name is the HTTP method and URL path, or just the method.
+    https://github.com/open-telemetry/opentelemetry-specification/pull/3165
+    https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/#name
+
```

Comment:
Check this change

Reply:
This seems like a potential breaking change:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1759

Reply:
Checking results of requests and dependencies. All columns match. So, I think we can consider this change to be okay.